targets/drive/mobile/main.jsx

```diff
@@ -77,7 +77,7 @@ const startApplication = async function(store, client) {
       // the server is not responding, but it doesn't mean we're revoked yet
       shouldInitBar = true
     }
-    else {
+    else if (store.getState().mobile.settings.serverUrl) {
```

Comment:
Prevents dispatching a `revoke` during the first boot, which would make you skip the onboarding.